ledger/src/blockstore_db.rs

```diff
@@ -415,13 +416,13 @@ pub(crate) struct Rocks {
 
 impl Rocks {
     pub(crate) fn open(path: PathBuf, options: BlockstoreOptions) -> Result<Rocks> {
-        let access_type = options.access_type.clone();
+        // let access_type = options.access_type.clone();
```

Comment:
Nit: Maybe remove this line?

Reply:
Ahh yes, I missed this; thank you! Addressed in https://github.com/anza-xyz/agave/pull/4214/commits/aed4738f5555f686238b419c85238ec2218881c3